openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -42,7 +42,6 @@ def _assert_error(self, data, expected_error, expected_error_description):
             json.loads(response.content),
             {u"error": expected_error, u"error_description": expected_error_description}
         )
-        self.assertNotIn("partial_pipeline", self.client.session)
```

Comment:
As above. Refactors aren't refactors if we don't keep test cases with the same end result - and the end result we want here is "there is no existing partial pipeline for this request".